suites/apis/submitFileTest.js

```diff
@@ -0,0 +1,191 @@
+let chai = require('chai');
+let chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+let expect = chai.expect;
+
+let assert = require('assert');
+
+
+Feature('SubmitFileTest');
+
+// Nodes, sorted hierarchically by key
+const nodes = [
```

Comment:
move nodes to a separated test data file because it is too large to put in a code files.